Bottom: 18760d732995b8379bc372377398a8fd58732762
Top:    08a4456877254843d89c18fe88e4c0431857fcff
Author: Alexandre Oliva <aoliva@redhat.com>
Date:   2013-08-13 22:08:59 -0300

Thread safety documentation.

for ChangeLog

	* manual/libdl.texi: New.


---

diff --git a/manual/libdl.texi b/manual/libdl.texi
new file mode 100644
index 0000000..e3fe045
--- /dev/null
+++ b/manual/libdl.texi
@@ -0,0 +1,10 @@
+@c FIXME these are undocumented:
+@c dladdr
+@c dladdr1
+@c dlclose
+@c dlerror
+@c dlinfo
+@c dlmopen
+@c dlopen
+@c dlsym
+@c dlvsym
